src/towncrier/templates/hr-between-versions.rst

```diff
@@ -1,22 +1,22 @@
 {% if render_title %}
 {% if versiondata.name %}
 {{ versiondata.name }} {{ versiondata.version }} ({{ versiondata.date }})
-{{ top_underline * ((versiondata.name + versiondata.version + versiondata.date)|length + 4)}}
+{{ top_underline * (get_underline_length(versiondata.name + versiondata.version + versiondata.date) + 4)}}
```

Comment:
One idea is a function like this

```diff
-{{ versiondata.name }} {{ versiondata.version }} ({{ versiondata.date }})
-{{ top_underline * (get_underline_length(versiondata.name + versiondata.version + versiondata.date) + 4)}}
+{{ top_section(versiondata.name, versiondata.version) }}
```

this function will join the arguments with a single space, and will generate 2 lines.

This looks to me less error prone

just an idea